WcaOnRails/spec/requests/regional_organizations_spec.rb

```diff
@@ -134,6 +134,7 @@
     context "when signed in as a board member and with invalid params" do
       it "renders the new regional organization form" do
         sign_in board_member
+        post new_regional_organization_path, params: { regional_organization: valid_attributes }
```

Comment:
What is this change doing? The context is called `context "when signed in as a board member and with invalid params" do` which seems to suggest that we should *not* post `valid_attributes` here.

Reply:
whoops - I think I accidentally added it, removed it now. Thanks for spotting!